jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -605,8 +606,12 @@ void java_bytecode_convert_methodt::convert(
   if((!m.is_abstract) && (!m.is_native))
   {
     code_blockt code(convert_parameter_annotations(m, method_type));
```

Comment:
Why is this line outside the if?

Reply:
Yes, it should be inside. Antonia is away for three days, so I've made a follow-up PR to make this change: https://github.com/diffblue/cbmc/pull/5121